purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -355,13 +355,7 @@ class Purchases internal constructor(
         storeProduct: StoreProduct,
         callback: PurchaseCallback,
     ) {
-        purchasesOrchestrator.startPurchase(
-            activity,
-            storeProduct.purchasingData,
-            null,
-            null,
-            callback,
-        )
+        purchase(PurchaseParams.Builder(activity, storeProduct).build(), callback)
```

Comment:
This is unrelated... but I saw that we were using methods that could be made internal in purchasesOrchestrator and took the opportunity to refactor it.